iphone/TitaniumKit/TitaniumKit/Sources/API/TiWindowProxy.m

```diff
@@ -46,6 +46,7 @@ - (void)_destroy
 
 - (void)_configure
 {
+  forceModal = YES;
```

Comment:
Why is this set to `YES` here when it's default value is defined as `NO`?

Reply:
Ah, I was just wondering what's the reason behind that. Thanks for the explanation.